music_assistant/providers/lastfm/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "plugin",
+  "domain": "lastfm",
+  "name": "LastFM Scrobbler",
+  "description": "Scrobble your music to Last.fm and others with a compatible API like Libre.fm",
+  "codeowners": ["@music-assistant"],
+  "documentation": "",
```

Comment:
```suggestion
  "documentation": "https://beta.music-assistant.io/plugins/lastfmscrobbler/",
```

Reply:
Just checking, will this still be correct with the rename I just did? `lastfm` => `lastfm_scrobble`

Reply:
let's keep the name of the docs the same as the domain